velox/docs/develop/aggregate-functions.rst

```diff
@@ -332,6 +353,43 @@ Finally, we implement the addSingleGroupIntermediateResults() method that is use
           const std::vector<VectorPtr>& args,
           bool mayPushdown) = 0;
 
+Finally, we can implement optional methods for converting raw inputs into intermediate
```

Comment:
there's another "finally" in line 336. Maybe remove the previous one to avoid repetition?